gprofiler/merge.py

```diff
@@ -92,33 +176,39 @@ def parse_perf_script(script: str):
             match = SAMPLE_REGEX.match(sample)
             if match is None:
                 raise Exception("Failed to match sample")
-            yield match.groupdict()
+            sample_dict = match.groupdict()
+
+            pid = int(sample_dict["pid"])
+            process_name = sample_dict["comm"]
+            stack = sample_dict["stack"]
+            if stack is not None:
```

Comment:
I'm not sure in which cases we get an empty stack. But the sample should still be accounted for the PID, otherwise we'll see skewed results. I think you should insert an empty stack in that case. I know it's from the previous state of the code, but let's fix it along the way.

CC @DanielShaulov

Reply:
That would only change it to a different misrepresentation - a few stacks that originate from different stacks but don't have for some reason the stack string would all be aggregated to the same value

Reply:
I didn't understand your comment lol.

My point here: sometimes, `perf` yields empty stacks for samples - without even 1 frame. I don't know why it happens. But the sample did happen - that process *did* execute at that time, so it should still be accounted....

Anyway, since this is here from before of your PR, I guess we can tackle it later.